packages/app/src/runs/modals/SelectCloudProjectModal.vue

```diff
@@ -85,6 +85,7 @@
             </span>
           </label>
           <a
+            v-if="projects.length > 0"
```

Comment:
Is this check ever falsy? If the length is 0, it doesn't look like this branch of markup would be generated.

Reply:
Do'h, Ryan filled me in. newProject is a `ref` and not `computed`, so this all works as expected.